programs/client/Client.cpp

```diff
@@ -528,8 +528,8 @@ void Client::connect()
                         << std::endl;
         }
     }
-
-    prompt_by_server_display_name = config().getRawString("prompt_by_server_display_name.default", "{display_name} :) ");
+    String connection_port = std::to_string(connection_parameters.port);
+    prompt_by_server_display_name = config().getRawString("prompt_by_server_display_name.default", "{display_name}") + ":" + connection_port +" :)";
```

Comment:
This is a format string that allows to configure the prompt string. Appending the port number to the format string is a bad idea. The right solution would be to change the default value of the format string to `{display_name}:{port}`. However, we cannot change the default value for everyone just because one user has an unusual environment.